core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -9838,6 +9838,24 @@ private void checkLiteral2(String expression, String expected) {
         .withHive().ok(expectedHive);
   }
 
+  /** Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-7050">[CALCITE-7050]
+   * Invalid unparse for FULL JOIN in MySQLDialect</a>. */
+  @Test void testFullJoin() {
```

Comment:
Maybe you should add another test. Dose the name like `testUnsupportedFullJoin`?